calcom/cal.com
# Fix 404 error when editing default availability & create new availability from event type page

## What does this PR do?



This PR fixes the 404 error when editing the default availability from an event type page. When editing the default availability a new availability will also be created linked to that event type.

Fixes #4608



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go to an event type with the default availability
- Go to the availability tab and click "Edit availability"
- You should be navigated to a new availability with the default named {eventTypeName} availability
- Check the DB or navigate back to the event type page to check that the new availability is connected to that event type

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
